src/main/java/de/keridos/floodlights/block/BlockGrowLight.java

```diff
@@ -206,9 +207,14 @@ public void breakBlock(World world, int x, int y, int z, Block block, int par6)
 
     @Override
     public void onBlockPlacedBy(World world, int x, int y, int z, EntityLivingBase entityLiving, ItemStack itemStack) {
-        if (world.getTileEntity(x, y, z) instanceof TileEntityFL) {
+        TileEntity tile = world.getTileEntity(x, y, z);
+        if (tile instanceof TileEntityFL) {
+            TileEntityFL tileFL = ((TileEntityFL) world.getTileEntity(x, y, z));
```

Comment:
```suggestion
        if (tile instanceof TileEntityFL) {
            TileEntityFL tileFL = ((TileEntityFL) tile);
```

Reply:
Or you could enable `enableModernJavaSyntax` in `gradle.properties`, which would allow for:
```suggestion
        if (tile instanceof TileEntityFL tileFL) {
```

Reply:
Missed zu remove. Sorry.